options/m_property.h

```diff
@@ -107,6 +107,9 @@ struct m_property_action_arg {
 };
 
 enum mp_property_return {
+    // Returned from validator if action should be executed.
```

Comment:
is this 'validator' concept an existing thing or did you just invent it?

Reply:
Yes, it is new concept.

Reply:
Just to expand, I wanted to avoid duplication of switch with the same cases for this type of property. So "validation" is a way to early exit when we don't actually going to return any data anyway.

Without this patch, single `mp.get_property_native`  produces 3 `VOCTRL` while 2 of them are completely unnecessary. And we does multiple properties we reduce the pressure by factor of 3.